cpp/src/DataStorm/ConnectionManager.cpp

```diff
@@ -63,7 +63,7 @@ ConnectionManager::remove(const Ice::ConnectionPtr& connection)
     exception_ptr ex;
     try
     {
-        connection->getInfo();
+        [[maybe_unused]] auto _ = connection->getInfo();
```

Comment:
I'm not following this code. Doesn't `getInfo` only throw if the connection is closed. Are we relying on this behavior to get the close exception?

Reply:
That's apparently what we do, yes. However, I am not following this code either.